.github/workflows/inspecjs-tests.yml

```diff
@@ -41,6 +44,12 @@ jobs:
 
      - name: Run unit tests
        run: yarn inspecjs test
+    
+     - name: Validate there are no changes between the inspec schemas and generated parsers
+       run: |
+        gem install inspec-bin -v 4.38.3
+        yarn inspecjs gen-types
+        yarn inspecjs validates-types
```

Comment:
`validates-types` is not working correctly, it is currently returning 0 even though the file are in fact modified on my local system:

```git
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   libs/inspecjs/schemas/exec-json.json
	modified:   libs/inspecjs/schemas/exec-jsonmin.json
	modified:   libs/inspecjs/schemas/profile-json.json
	modified:   libs/inspecjs/src/generated_parsers/v_1_0/exec-json.ts
	modified:   libs/inspecjs/src/generated_parsers/v_1_0/exec-jsonmin.ts
	modified:   libs/inspecjs/src/generated_parsers/v_1_0/profile-json.ts
```

```bash
➜  heimdall2 git:(automateQuicktypeInspecJS) ✗ git diff --quiet HEAD master -- libs/inspecjs/schemas libs/inspecjs/src/generated_parsers/
➜  heimdall2 git:(automateQuicktypeInspecJS) ✗ echo $?
0
```

I think the diff command needs to be

```
git diff --quiet HEAD -- libs/inspecjs/schemas libs/inspecjs/src/generated_parsers/ 
```

for debugging purposes if you remove the `--quiet` you can see what it is actually returning.